.github/RELEASE_TEMPLATE.md

```diff
@@ -1,150 +1,89 @@
-# Overview
+Compatible with [`jormungandr@{{JORM_TAG}}`](https://github.com/input-output-hk/jormungandr/releases/tag/{{JORM_TAG}}) and [`cardano-node@{{CARDANO_NODE_TAG}}`](https://github.com/input-output-hk/cardano-node/releases/tag/{{CARDANO_NODE_TAG}}).
 
-<!-- Short high-level description about the content of the release.
-This should also include a short presentation of files that are included, 
-the target operating systems etc ... -->
-
-| Artifact                                                | Platform                  | Description                                                  |
-| ---                                                     | ---                       | ---                                                          |
-| `cardano-wallet-jormungandr@vYYYY-MM-DD.linux64.sha256` | <pre>Linux 64-bit</pre>   | Checksum for the Linux executable.                           |
-| `cardano-wallet-jormungandr@vYYYY-MM-DD.linux64.tar.gz` | <pre>Linux 64-bit</pre>   | Linux executable with auto-completion script for bash shell. |
-| `cardano-wallet-jormungandr@vYYYY-MM-DD.macos64.sha256` | <pre>MacOs 64-bit</pre>   | Checksum for the MacOS executable.                           |
-| `cardano-wallet-jormungandr@vYYYY-MM-DD.macos64.tar.gz` | <pre>MacOs 64-bit</pre>   | MacOS executable with necessary system libraries.            |
-| `cardano-wallet-jormungandr@vYYYY-MM-DD.win64.sha256`   | <pre>Windows 64-bit</pre> | Checksum for Windows executable.                             |
-| `cardano-wallet-jormungandr@vYYYY-MM-DD.win64.zip`      | <pre>Windows 64-bit</pre> | Windows executable with necessary DLLs.                      |
-
-# Installation Instruction
-
-<!-- Specific installation steps for this particular release. This should
-basically captures whatever is currently available on the repository at
-the moment of releasing. --> 
-
-## Linux 64-bit
-
-1. Install [jormungandr@X.Y.Z](https://github.com/input-output-hk/jormungandr/releases/tag/vX.Y.Z) from the official repository.
-
-2. Download `cardano-wallet-jormungandr@vYYYY-MM-DD.linux64.tar.gz` and uncompress it in a directory that is on your `$PATH`, e.g. `/usr/local/bin`.
-
-  ```bash
-  $ curl -L https://github.com/input-output-hk/cardano-wallet/releases/download/vYYYY-MM-DD/cardano-wallet-jormungandr@vYYYY-MM-DD.linux64.tar.gz | tar xz -C $HOME/.local/bin
-  ```
-
-3. (optional) Install the bash/zsh auto-completion script:
-
-  ```bash
-  $ mv $HOME/.local/bin/cardano-wallet.sh > /etc/bash_completion.d/cardano-wallet.sh
-  $ source /etc/bash_completion.d/cardano-wallet.sh
-  ```
-
-4. Start `cardano-wallet --help` and see available parameters.
-
-#### TroubleShooting
-
-<details>
-  <summary>cardano-wallet: error while loading shared libraries: <code>libcrypto.so.1.0.0</code>: cannot open shared object file: No such file or directory</summary>
```

Comment:
> One question though, why did you remove the "TroubleShooting" section and the bits about libcrypto?
Was this moved to the wiki?

@KtorZ This section was in the template, but I don't think it was ever used in an actual release. I started with replacing the template with the most recently used release-notes.

I guess we could add it?

Reply:
Re-added.